src/main/resources/application.properties

```diff
@@ -5,4 +5,14 @@ spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 spring.h2.console.enabled=true
 spring.jpa.defer-datasource-initialization=true
-server.port=8080
\ No newline at end of file
+server.port=8085
+management.endpoints.web.exposure.include= info,beans,configprops,mappings,shutdown,health,metrics
```

Comment:
Combine all similar properties together with proper spaces so that you will not get confuse later when you see it